docs/web_api.md

```diff
@@ -1607,6 +1607,59 @@ Currently we support Slic3r derivatives and Cura with Cura-Octoprint.
   }
   ```
 
+## History APIs
+The APIs below are avilable when the `[history]` plugin has been configured.
+
+### Get job list
+- HTTP command:\
+  `GET /printer/histor/list?limit=50&start=50&since=1&before=5`
```

Comment:
This endpoint is now on the "/server/" path and history is missing a "y". 

It appears that the path needs to be updated for all of the HTTP and websocket endpoints, so I'll only add comment here.